combine-packages-coverage.js

```diff
@@ -44,15 +44,6 @@ const nextStream = function () {
 	return fs.createReadStream(file);
 };
 
-// first prepend the base path of each package ('packages/stark-xxxx') to every source file reference in the coverage file
-// this ensures that Coveralls can show the list of packages correctly including their files
-for (const fileName of fileNames) {
-	const packageName = fileName.match(/stark-\w*/)[0];
-	const replacements = [{ searchValue: /SF:(.*)(\r|\n)/, replaceValue: `SF:packages/${packageName}/$1$2` }];
```

Comment:
Are you sure that with `ng test` the reports are split by package name? So that the coverage of all stark packages is shown in Coveralls correctly

Reply:
Actually, since we use `ng test stark-core --code-coverage` from stark root folder instead of `ng test` from "packages/stark-core" folder, the paths are correct.

`ng test --code-coverage` from "packages/stark-core", lcov contains path "SF:src/common/error/error.ts"
`ng test stark-core --code-coverage` from root, lcov contains path "SF:packages/stark-core/src/common/error/error.ts"